graph/edge/impl/TypeEdgeImpl.java

```diff
@@ -344,6 +379,69 @@ public void commit() {
      */
     public static class Persisted extends TypeEdgeImpl implements TypeEdge {
 
+        public static class EncodedValue {
+            private static final byte KEY_OVERRIDEN = 1;
```

Comment:
We store the information that an OWNS edge has the unique constraint in the RocksDB value for the TypeEdge.
Originally, this value was only used for the overriden type (`owns concrete-attribute AS abstract-attributes`).

We convert the value into a key-value list. Thankfully, the overriden bytes were vertices, so they have a prefix. 
We recognise & handle this case for backward compatibility, but we insert new ones with KEY_OVERRIDEN as a key.

This can be extended to include cardinality information.